src/cr/cube/cubepart.py

```diff
@@ -775,9 +775,14 @@ def pairwise_significance_tests(self):
 
     @lazyproperty
     def population_counts(self):
```

Comment:
Even if the porperty doesn't have a doctring, considering that you touched it you should provide a meaningful docstring also because it introduces new business logic

Reply:
hehe, this is a revenge I see :)

Reply:
ahahaha no it is not :D it's steve's lessons

Reply:
❤️

Reply:
Addressed